libraries/core/src/main/kotlin/com/bumble/appyx/core/node/Node.kt

```diff
@@ -100,6 +100,11 @@ open class Node(
         )
     }
 
+    @Deprecated("Will be removed in 1.1. Use executeAction instead")
```

Comment:
You can add `replaceWith = ReplaceWith("executeAction(action)")` so IDEA can replace it automatically.